Source/SimConnect/Source/DataDefinitions/SimConnectDataDefinitionId.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2024 Steven Frost and Orion Lyau
+
+#pragma once
+
+#include <MSFS/MSFS_WindowsTypes.h>
+#include <SimConnect.h>
+
+// -----------------------------------------------------------------------------
+
+namespace SimConnect
+{
+
+// -----------------------------------------------------------------------------
+
+enum SimConnectDataDefinitionId : SIMCONNECT_DATA_DEFINITION_ID
+{
+	AircraftPosition // AircraftPositionDataDefinition
```

Comment:
Do we want to do something so we don't have `AircraftPosition` floating around directly in the SimConnect namespace (since it's an `enum` and not `enum class`)?

Reply:
Yeah good shout, will convert this to an `enum class` & add a utility to get the value from it.